run.py

```diff
@@ -0,0 +1,172 @@
+from flask import render_template, request, jsonify, Flask
+import os, re, datetime
+from app import db
+from app.product.models import ProductModel
+
+
+app = Flask(__name__)
+BASE_URL = '/products'
+
+if not os.path.isfile('products.db'):
+    db.connect()
+
+def isValid(email):
+    regex = re.compile(r'([A-Za-z0-9]+[.-_])*[A-Za-z0-9]+@[A-Za-z0-9-]+(\.[A-Z|a-z]{2,})+')
+    if re.fullmatch(regex, email):
+      return True
+    else:
+      return False
+
+@app.route(BASE_URL, methods=["POST"])
+def postProduct():
+   req_data = request.get_json()
+   ############################
+   product_name = req_data['product_name']
+   category = req_data['category']
+   producer = req_data['producer']
+   description = req_data['description']
+   price = req_data['price']
+   prd = [p.serialize() for p in db.view()]
+   for p in prd:
```

Comment:
`p` i `prd` to nie są najlepsze nazwy. Powinno się unikać nazw, które nie mają znaczenia

Reply:
git